src/util.py

```diff
@@ -0,0 +1,44 @@
+import errno
+import re
+import socket
+
+from enum import Enum
+from socket import error as socket_error
+
+
+# regex
+num_regex = re.compile(r"(\d+)")
+
+
+# load the ip address from the running machine
+# Reference: https://github.com/yahoo/TensorFlowOnSpark/blob/e2f5cc45f95812d163e75b6ddb9c4661261d3bb0/tensorflowonspark/util.py#L41
+def get_ip_address():
+  """Simple utility to get host IP address."""
+  try:
+    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    s.connect(("8.8.8.8", 80))
+    ip_address = s.getsockname()[0]
+  except socket_error as sockerr:
+    if sockerr.errno != errno.ENETUNREACH:
+      raise sockerr
+    ip_address = socket.gethostbyname(socket.getfqdn())
+  finally:
+    s.close()
+
+  return ip_address
+
+
+def config_input_check(text:str, available:int, res:Enum):
+
+    if text is "" or not num_regex.match(text):
```

Comment:
This can be `if not text or not num_regex.match(text):`. This will prevent errors popping up if `text` is a byte string or similar (if that's even possible here).

Reply:
fixed.